src/backend_stub/config.js

```diff
@@ -3,8 +3,19 @@
 var webpackConfig = require('../../webpack.config.js');
 
 module.exports = {
-	httpBaseUrl: '',
 	httpPort: 3001,
-	staticPort: 3002,
-	staticPath: webpackConfig.output.path
+	staticBaseUrl: webpackConfig.output.publicPath,
+	staticPath: webpackConfig.output.path,
+	dataHostnames: [
+		'd0.knowsheet.local',
```

Comment:
I'd appreciate a comment as per what should one do with these hostnames on localhost and in prod.